docs/reference/api/docker_remote_api.md

```diff
@@ -66,7 +66,7 @@ without protocol. Throughout this structure, double quotes are required.
 If you are using `docker-machine`, the Docker daemon is on a virtual host that uses an encrypted TCP socket. This means, for Docker Machine users, you need to add extra parameters to `curl` or `wget` when making test API requests, for example:
 
 ```
-curl --insecure --cert ~/.docker/cert.pem --key ~/.docker/key.pem https://YOUR_VM_IP:2376/images/json
+curl --insecure --cert %USERPROFILE%\.docker\machine\machines\<name_of_your_machine>\cert.pem --key %USERPROFILE%\.docker\machine\machines\<name_of_your_machine>\key.pem https://YOUR_VM_IP:2376/images/json
```

Comment:
This is now targeted at Windows only (I don't have access to a Windows machine, so haven't tested it).
Perhaps we should either have instructions for Windows separately, or use linux paths (same as the `wget` instructions below).

(I think for OS X, more actions were required to get it working, because the keys were in a format that OS X had trouble with)